kernel/ovpn-dco/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2008-2010 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=ovpn-dco
+PKG_RELEASE:=$(AUTORELEASE)
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/OpenVPN/ovpn-dco.git
+PKG_SOURCE_DATE:=2021-10-05
+PKG_SOURCE_VERSION:=1017d4ada58a4c73f44ef671b6469ae168b6c6b0
+PKG_MIRROR_HASH:=5938ca14c7b3235b60da227f43638556272dea325ddb658bce5e0dadaebb923e
+
+PKG_LICENSE:=GPL-2.0
+
+PKG_MAINTAINER:=Jianhui Zhao <zhaojh329@gmail.com>
```

Comment:
There is missing some order. You should take a look at others Makefile in this repository.